tests/unit/test_clidocs.py

```diff
@@ -407,6 +408,41 @@ def test_includes_global_args_ref_in_html_options(self):
             "global parameters", rendered
         )
 
+    def test_includes_generate_cli_skeleton_note_in_man_options(self):
```

Comment:
If we add it directly to the docstring of the argument, we can go ahead and remove these tests. We don't need tests for direct docstrings.